azext_iot/tests/test_iot_ext_unit.py

```diff
@@ -820,13 +827,23 @@ def test_config_create_invalid_args(self, serviceclient, req, arg):
                                                        priority=req['priority'], labels=req['labels'])
 
     @pytest.mark.parametrize("req", [
-        (generate_device_config(content_type='device'))
+        (generate_device_config(content_type='device')),
```

Comment:
seems like a trailing comma

Reply:
Yea the trailing commas are coming from auto-formatting using Python Black...it's an amazing tool.